src/Components/PresentationalComponents/CVEPageDetails/CVEPageDetails.js

```diff
@@ -18,22 +18,15 @@ const CVEPageDetails = ({ data, changeExposedSystemsParameters }) => {
                         </WithLoader>
                     </GridItem>
 
-                    <GridItem md={4} sm={12} className="severity-info-box pf-u-mt-md">
-                        <CVEPageDetailsSeverity
-                            impact={data.data.impact}
-                            cvss2_score={data.data.cvss2_score}
-                            cvss3_score={data.data.cvss3_score}
-                            cvss2_metrics={data.data.cvss2_metrics}
-                            cvss3_metrics={data.data.cvss3_metrics}
-                        />
+                    <GridItem md={4} sm={12} className="pf-u-mt-sm pf-u-ml-sm">
+                        <CVEPageDetailsSeverity cveAttributes={data.data} />
```

Comment:
what do you think about a better naming? cause it contains info about business risk and status id